packages/zowe-explorer/__tests__/__unit__/extending/ZoweExplorerExtender.unit.test.ts

```diff
@@ -59,6 +59,7 @@ describe("ZoweExplorerExtender unit tests", () => {
         Object.defineProperty(fs, "existsSync", { value: newMocks.mockExistsSync, configurable: true });
         jest.spyOn(Profiles, "getInstance").mockReturnValue(newMocks.profiles);
         jest.spyOn(ZoweExplorerExtender.prototype, "getProfilesCache").mockReturnValue(newMocks.profiles);
+        newMocks.profiles.resolveTypePromise = jest.fn(() => Promise.resolve());
```

Comment:
Should we update the utility method `createInstanceOfProfile` that's invoked in this test, to reduce code duplication for future tests that may rely on `resolveTypePromise`?